### PR TITLE
HOCS-3344 Add notify to docker-compose.yml

### PR DESCRIPTION
**Pull request by fewtrees-ho**
https://collaboration.homeoffice.gov.uk/jira/browse/HOCS-3344

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -357,7 +357,39 @@ services:
         condition: service_started
       keycloak:
         condition: service_started
+      notify:
+        condition: service_started
 
+  notify:
+    healthcheck:
+      test: ["CMD", "java", "/app/scripts/HealthCheck.java", "http://localhost:8080/actuator/health", "||", "exit", "1"]
```
Comment (LiamMacP) [inline docker/docker-compose.yml:365]: Just incase you didn't know - this file doesn't exist in notify at present 🙂

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -357,7 +357,39 @@ services:
...
+      start_period: 3m
+      timeout: 30s
+      retries: 3
+    image: quay.io/ukhomeofficedigital/hocs-notify:${INFO_TAG:-latest}
```
Comment (LiamMacP) [inline docker/docker-compose.yml:369]:
```suggestion
    image: quay.io/ukhomeofficedigital/hocs-notify:${NOTIFY_TAG:-latest}
```

File: docker/docker-compose.yml (hunk 1 above)
Comment (fewtrees-ho) [reply docker/docker-compose.yml:369]: awesome!

File: docker/docker-compose.yml (hunk 2 above)
Comment (fewtrees-ho) [reply docker/docker-compose.yml:373]: Spotted that and updated... ..already..

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**